lib/jobs.py

```diff
@@ -53,3 +54,49 @@ async def add_job(job_dict):
 
     with litani.atomic_write(jobs_dir / ("%s.json" % job_id)) as handle:
         print(json.dumps(job_dict, indent=2), file=handle)
+
+
+def get_jobs():
+    out = []
+    jobs_dir = litani.get_cache_dir() / litani.JOBS_DIR
+    for job in jobs_dir.iterdir():
+        with open(job) as handle:
+            job_dict = json.load(handle)
+            for key in ("job_id", "status_file", "subcommand"):
+                job_dict.pop(key)
+            out.append(job_dict)
+    return out
+
+
+def print_jobs(_=None):
+    jobs = get_jobs()
+    print(json.dumps(jobs, indent=2))
+    sys.stdout.flush()
+    os.close(sys.stdout.fileno())
```

Comment:
I realise I didn't write a comment here originally. Would you mind explaining that `sys.stdout.close()` doesn't work because it only closes the Python object representing the file, but the underlying file handle (in the C stdlib) for std{in,out,err} would remain open, which is not what we want.